src/Aspire.Hosting/DistributedApplicationExecutionContext.cs

```diff
@@ -13,4 +13,14 @@ public class DistributedApplicationExecutionContext(DistributedApplicationOperat
     /// The operation currently being performed by the AppHost.
     /// </summary>
     public DistributedApplicationOperation Operation { get; } = operation;
+
+    /// <summary>
+    /// Returns true if the current operation is publishing.
+    /// </summary>
+    public bool IsPublishing => Operation == DistributedApplicationOperation.Publish;
+
+    /// <summary>
+    /// Returns true if the current operation is running.
+    /// </summary>
+    public bool IsRunning => Operation == DistributedApplicationOperation.Publish;
```

Comment:
"IsRunning" is kind of a weird name. The app isn't actually running yet.

"IsRunContext" ?

Reply:
IsRunMode / IsPublishMode?

Reply:
I like IsRunMode more than IsRunning.

Reply:
Mode hmmmm, don't love it but....

Reply:
`IsRunning` that bad? I don't have a strong opinion here. Although I find it weird that we have `.Operation` but a `xMode` suffix on the boolean properties.